src/components/DisplayArticle.jsx

```diff
@@ -1,33 +1,31 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getArticles } from "../modules/getArticles";
 import { Container } from "semantic-ui-react";
+import { useDispatch, useSelector } from "react-redux";
 
 const DisplayArticle = () => {
-  const [article, setArticle] = useState();
-  const [message, setMessage] = useState();
+  const dispatch = useDispatch();
+  const viewArticle = useSelector((state) => state.specificArticle);
+  const errorMessage = useSelector((state) => state.errorMessage);
   const { id } = useParams();
-  useEffect(() => {
-    const getSpecificArticle = async () => {
-      const response = await getArticles.show(id);
-      response.id ? setArticle(response) : setMessage(response);
-    };
 
-    getSpecificArticle();
+  useEffect(() => {
+    getArticles.show(id, dispatch);
   }, [id]);
 
   return (
     <>
-      {article && (
+      {viewArticle && (
```

Comment:
Not sure about the conditional here, but let it be for now...

Reply:
Yes, we will probably fix this further down the line :)